backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/storage/disk/image/CopyManagedBlockDiskCommand.java

```diff
@@ -167,7 +167,7 @@ protected void createVolumes() {
         // Attach source to host (if source is Managed Block Storage)
         // TODO: handle failures
         if (sourceDomainType == StorageDomainType.ManagedBlockStorage) {
-            String sourcePath = attachVolume(createManagedBlockDiskFromDiskImage(sourceDisk));
```

Comment:
so how about renaming `createManagedBlockDiskFromDiskImage` to something like `createDestinationManagedBlockDiskFromSourceDiskImage`? a bit longer but would make it more clear that it's not an equivalent to casting..

Reply:
or maybe `toDestinationManagedBlockDisk(DiskImage sourceImage)`?

Reply:
I agree although I am not sure about the naming, maybe a better way would be to have `createManagedBlockDiskFromDiskImage` do the casting if object is already a `ManagedBlockDisk` but that could be confusing too